prime-angular-frontend/src/app/shared/components/dialogs/content/claim-enrollee/claim-enrollee.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+import { AdjudicationResource } from '@adjudication/shared/services/adjudication-resource.service';
+import { Admin } from '@auth/shared/models/admin.model';
+import { BehaviorSubject } from 'rxjs';
+import { Enrolment } from '@shared/models/enrolment.model';
+import { MatDialogRef } from '@angular/material';
+import { ConfirmDialogComponent } from '../../confirm-dialog/confirm-dialog.component';
+
+export class ClaimEnrolleeAction {
+  public action: ClaimActionEnum;
+  public adjudicatorId?: number;
+}
+
+export enum ClaimActionEnum {
+  UnClaim = 0,
+  Claim = 1,
+}
+
+@Component({
+  selector: 'app-claim-enrollee',
+  templateUrl: './claim-enrollee.component.html',
+  styleUrls: ['./claim-enrollee.component.scss']
+})
+export class ClaimEnrolleeComponent implements OnInit {
+  public adjudicators: BehaviorSubject<Admin[]>;
```

Comment:
Convention for Observables is to suffix them with a $ - `adjudicators$`.  So everyone knows immediately that they are dealing with a stream